xml/ha_cluster_lvm.xml

```diff
@@ -117,6 +117,24 @@
      otherwise lvmlockd will fail to start.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     On all nodes, check the value of <parameter>use_lvmlockd</parameter>
+     in <filename>/etc/lvm/lvm.conf</filename>:
+     </para>
+     <screen>&prompt.root;grep 'use_lvmlockd =' /etc/lvm/lvm.conf</screen>
+     <para>
+      If the command returns <literal>use_lvmlockd = 0</literal>, set the
+      parameter's value to <literal>1</literal> on <emphasis>all</emphasis> nodes:
+     </para>
+     <screen>&prompt.root;sed -ie 's/use_lvmlockd = 0/use_lvmlockd = 1/' /etc/lvm/lvm.conf</screen>
```

Comment:
The `sed` expression is dangerous. Again, the whitespace issue as before: If the user writes `use_lvmlockd=0`, the regex won't match.

As such, I would correct the above expression and would use:

```suggestion
     <screen>&prompt.root;sed -ie 's/use_lvmlockd\s*=\s*0/use_lvmlockd = 1/' /etc/lvm/lvm.conf</screen>
```

I've tested it as follows:

```
$ cat sedtest 
# Hello
use_lvmlockd = 0
use_lvmlockd=0
use_lvmlockd= 0

$ sed -e 's/use_lvmlockd\s*=\s*0/use_lvmlockd = 1/' sedtest
# Hello
use_lvmlockd = 1
use_lvmlockd = 1
use_lvmlockd = 1
```

Reply:
Hi, 

If I  may add a comment here, I used `sed` command just as an example. 
@taroth21 Please feel free to reformat. I was just focusing on enabling `use_lvmlockd` parameter in `/etc/lvm/lvm.conf` configuration file.

Best regards,
Ab

Reply:
The `sed` command is fine! :+1: It's just the regular expression that doesn't match all possible cases. Probably, 99% of users wouldn't be affected, but with my simple change above, we can catch them all. :wink:

Reply:
Agree. :)